UnitTests/Drawing/CellTests.cs

```diff
@@ -1,78 +1,78 @@
     {
-        var rc = new RuneCell ();
-        Assert.NotNull (rc);
-        Assert.Equal (0, rc.Rune.Value);
-        Assert.Null (rc.ColorScheme);
+        var c = new Cell ();
+        Assert.True (c is { });
+        Assert.Equal (0, c.Rune.Value);
+        Assert.Null (c.Attribute);
     }
 
     [Fact]
     public void Equals_False ()
     {
-        var rc1 = new RuneCell ();
+        var c1 = new Cell ();
 
-        var rc2 = new RuneCell
+        var c2 = new Cell
         {
-            Rune = new ('a'), ColorScheme = new() { Normal = new (Color.Red) }
+            Rune = new ('a'), Attribute = new (Color.Red)
         };
-        Assert.False (rc1.Equals (rc2));
-        Assert.False (rc2.Equals (rc1));
-
-        rc1.Rune = new ('a');
-        rc1.ColorScheme = new ();
-        Assert.Equal (rc1.Rune, rc2.Rune);
-        Assert.False (rc1.Equals (rc2));
-        Assert.False (rc2.Equals (rc1));
+        Assert.False (c1.Equals (c2));
+        Assert.False (c2.Equals (c1));
+
+        c1.Rune = new ('a');
+        c1.Attribute = new ();
+        Assert.Equal (c1.Rune, c2.Rune);
+        Assert.False (c1.Equals (c2));
+        Assert.False (c2.Equals (c1));
     }
 
     [Fact]
     public void Equals_True ()
     {
-        var rc1 = new RuneCell ();
-        var rc2 = new RuneCell ();
-        Assert.True (rc1.Equals (rc2));
-        Assert.True (rc2.Equals (rc1));
-
-        rc1.Rune = new ('a');
-        rc1.ColorScheme = new ();
-        rc2.Rune = new ('a');
-        rc2.ColorScheme = new ();
-        Assert.True (rc1.Equals (rc2));
-        Assert.True (rc2.Equals (rc1));
+        var c1 = new Cell ();
+        var c2 = new Cell ();
+        Assert.True (c1.Equals (c2));
+        Assert.True (c2.Equals (c1));
+
+        c1.Rune = new ('a');
+        c1.Attribute = new ();
+        c2.Rune = new ('a');
+        c2.Attribute = new ();
+        Assert.True (c1.Equals (c2));
+        Assert.True (c2.Equals (c1));
     }
 
     [Fact]
     [AutoInitShutdown (configLocation: ConfigurationManager.ConfigLocations.DefaultOnly)]
-    public void RuneCell_LoadRuneCells_InheritsPreviousColorScheme ()
+    public void Cell_LoadCells_InheritsPreviousAttribute ()
     {
-        List<RuneCell> runeCells = new ();
+        List<Cell> cells = [];
 
         foreach (KeyValuePair<string, ColorScheme> color in Colors.ColorSchemes)
         {
             string csName = color.Key;
 
             foreach (Rune rune in csName.EnumerateRunes ())
             {
-                runeCells.Add (new() { Rune = rune, ColorScheme = color.Value });
+                cells.Add (new() { Rune = rune, Attribute = color.Value.Normal });
             }
 
-            runeCells.Add (new() { Rune = (Rune)'\n', ColorScheme = color.Value });
+            cells.Add (new() { Rune = (Rune)'\n', Attribute = color.Value.Focus });
         }
 
         TextView tv = CreateTextView ();
-        tv.Load (runeCells);
+        tv.Load (cells);
```

Comment:
Is it possible to refactor this test to not use `TextView`. IMO, unit tests for a class hsould not depend on any other classes.

If this is really a TV test, then move it to TV tests.